cmd/podmanV2/containers/top.go

```diff
@@ -0,0 +1,91 @@
+package containers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/psgo"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	topDescription = fmt.Sprintf(`Similar to system "top" command.
+
+  Specify format descriptors to alter the output.
+
+  Running "podman top -l pid pcpu seccomp" will print the process ID, the CPU percentage and the seccomp mode of each process of the latest container.
+  Format Descriptors:
+    %s`, strings.Join(psgo.ListDescriptors(), ","))
+
+	topOptions = entities.TopOptions{}
+
+	topCommand = &cobra.Command{
+		Use:   "top [flags] CONTAINER [FORMAT-DESCRIPTORS|ARGS]",
+		Short: "Display the running processes of a container",
+		Long:  topDescription,
+		RunE:  top,
+		Args:  cobra.ArbitraryArgs,
+		Example: `podman top ctrID
+podman top --latest
+podman top ctrID pid seccomp args %C
+podman top ctrID -eo user,pid,comm`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: topCommand,
+	})
+
```

Comment:
Because top is a direct child of rootCmd

```suggestion
	topCommand.SetHelpTemplate(registry.HelpTemplate())
	topCommand.SetUsageTemplate(registry.UsageTemplate())
```